app/domain/authentication/authn_jwt/signing_key/create_jwks_from_http_response.rb

```diff
@@ -4,9 +4,10 @@ module SigningKey
       # CreateJwksFromHttpResponse command class is responsible to create jwks object from http response
       CreateJwksFromHttpResponse ||= CommandClass.new(
         dependencies: {
-          logger: Rails.logger
+          logger: Rails.logger,
+          jwk_set: JSON::JWK::Set
```

Comment:
i would call this `jwk_set_class` as you call its `.new` method later to create an instance

Reply:
Done